pyproject.toml

```
@@ -1,3 +1,48 @@
+[project]
+name = "pycfmodel"
+version = "1.1.0"
+readme = "README.md"
+description="A python model for Cloud Formation scripts"
+requires-python=">=3.9.0"
+dependencies = [
+    "pydantic~=2.7"
+]
+
+[project.urls]
+Homepage = "https://github.com/Skyscanner/pycfmodel"
+
+[[project.authors]]
+name = "Skyscanner Product Security"
```

This is more accurate to current teams:
```suggestion
name = "Skyscanner Security"
```